.idea/dataSources.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
This shouldn't be here, is better to put the `.idea` folder and every file related to the IDE in the `.gitignore`

Reply:
I'll fix it, any other file that should be in `.gitignore`?